org.adempiere.base/src/org/adempiere/impexp/GridTabCSVExporter.java

```diff
@@ -455,6 +463,46 @@ private Object resolveValue(GridTab gridTab, MTable table, MColumn column, int i
 		return value;
 	}
 	
+	/**
+	 * @param gridTab
+	 * @param table
+	 * @param column
+	 * @param i row index
+	 * @param headName
+	 * @return column value
+	 */
+	private Object resolveValueMultiSelection(GridTab gridTab, MTable table, MColumn column, int i, String headName) {
+		Object retValue = null;
+		if (headName.indexOf("[") >= 0 && headName.endsWith("]")) {
+			String foreignTable = column.getMultiReferenceTableName();
+			Object idO = gridTab.getValue(i, column.getColumnName());
+			if (idO != null && idO instanceof String) {
+					List<Object> params = new ArrayList<>();
+					String[] values = ((String)idO).split(",");
+					int start = headName.indexOf("[")+1;
+					int end = headName.length()-1;
+					String foreignColumn = headName.substring(start, end);
+					StringBuilder select = new StringBuilder("SELECT ")
+							.append(foreignColumn).append(" FROM ")
+							.append(foreignTable).append(" WHERE ")
+							.append(foreignTable).append("_ID IN (");
+					
+					for(int idx=0;idx<values.length;idx++) {
+						if(idx==values.length-1)
+							select.append("?)");
+						else
+							select.append("?,");
+						params.add(Integer.valueOf(values[idx]));
+					}
+					
+					retValue = DB.getSQLValueStringConcatMultipleResultsEx(null,select.toString(),params);

```

Comment:
The problem with this approach is when the column contains commas, then you would end up with a corrupt CSV file.